fast/stages/0-bootstrap/variables.tf

```diff
@@ -85,6 +91,30 @@ variable "custom_roles" {
   default     = {}
 }
 
+variable "environments" {
+  description = "Environment names."
+  type = map(object({
+    name       = string
+    is_default = optional(bool, false)
+  }))
+  nullable = false
+  default = {
+    dev = {
+      name = "Development"
```

Comment:
- Why the nested map? are you expecting more attributes in the future?
- Can you have more than one default? Would it make more sense to have the default outside? Something like this:
  ```terraform
  type = object({
    names = map(string)
    default = string
  })
  ```
  This also removes the need for a check

Reply:
but then you need to be consistent, and I would still need to check that default is in names...

TBH I prefer the idea of an environment as "an object", like we do for everything else in modules